Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValueImplementation/Rendering/BocAutoCompleteReferenceValueRenderer.cs

```diff
@@ -375,7 +364,17 @@ private TextBox GetTextBox (BocRenderingContext<IBocAutoCompleteReferenceValue>
       textBox.Page = renderingContext.Control.Page!.WrappedInstance;
       textBox.ApplyStyle(renderingContext.Control.CommonStyle);
       renderingContext.Control.TextBoxStyle.ApplyStyle(textBox);
+
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Combobox);
       textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaAutoComplete, HtmlAriaAutoCompleteAttributeValue.Both);
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaExpanded, HtmlAriaExpandedAttributeValue.False);
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaHasPopup, HtmlAriaHasPopupAttributeValue.True);
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaControls, GetDropDownButtonName(renderingContext));
```

Comment:
this one is new. please explain why the textbox has a controls relationship with the button.

Reply:
That's just a mistake that is fixed by the JavaScript code :p it should reference the result set instead

Reply:
So, I found a mistake in the review that will get fixed :) It was simply compensated for already and now should also get cleaned up. 
(just summarizing to make sure we are talking about the same here)